src/libcore/str/mod.rs

```diff
@@ -2347,9 +2336,7 @@ impl str {
     }
 
     /// Converts a string slice to a byte slice. To convert the byte slice back
-    /// into a string slice, use the [`str::from_utf8`] function.
-    ///
-    /// [`str::from_utf8`]: ./str/fn.from_utf8.html
+    /// into a string slice, use the [`from_utf8`] function.
```

Comment:
Let's keep the same link text.
```suggestion
    /// into a string slice, use the [`str::from_utf8`](from_utf8) function.
```